### PR TITLE
Mesh_3 - add an example for `Lipschitz_sizing`

### PR DESCRIPTION
**Pull request by janetournois**
## Summary of Changes

This PR introduces an example about how to use the simple `CGAL::Lipschitz_sizing`, that is in the experimental section of `Mesh_3`. This example should be improved in the next release, once polyhedral domains will be enriched.

## Release Management

* Affected package : Mesh_3
* Issue(s) solved (if any): none
* Feature/Small Feature (if any): no

### CODE REVIEW

**Comment by lrineau** [comment]
There is a warning in the Doxygen testsuite:
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-4.10-Ic-144/Mesh_3.log
```
.../CGAL-4.10-Ic-144/doc/Mesh_3/Mesh_3.txt:171: Warning: unable to resolve reference to `Mesh_3/mesh_polyhedral_domain_with_lipschitz_sizing.cpp' for \ref command
```

**Comment by janetournois** [comment]
sorry!

File: Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+//
+// Author(s)     : Jane Tournois
+//
+//******************************************************************************
+// File Description :
+//
+//
+//******************************************************************************
+
+#ifndef CGAL_LIP_SIZING_PARAMETERS
```
Comment (sloriot) [inline Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h]: include guard should end with _H

File: Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_polyhedron.h [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+// Author(s)     : Jane Tournois
+//
+//******************************************************************************
+// File Description :
+//
+//
+//******************************************************************************
+
+
+#ifndef _LIPSCHITZ_SIZING_POLYHEDRON_
```
Comment (sloriot) [inline Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_polyhedron.h]: `CGAL_LIPSCHITZ_SIZING_POLYHEDRON_H`

File: Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+#define CGAL_LIP_SIZING_PARAMETERS
+
```
Comment (sloriot) [inline Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h]: you're missing `#include <CGAL/Mesh_3/licence.h>`

File: Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_polyhedron.h [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+//
+//
+// Author(s)     : Jane Tournois
+//
+//******************************************************************************
```
Comment (sloriot) [inline Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_polyhedron.h]: needed if empty

File: Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+//
+//
+// Author(s)     : Jane Tournois
+//
+//******************************************************************************
```
Comment (sloriot) [inline Mesh_3/include/CGAL/Mesh_3/experimental/Lipschitz_sizing_parameters.h]: not needed if empty

**Pull request merged**